 <!DOCTYPE html>
<html>
<head>
    <link rel="stylesheet" href="layout.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
    <link href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
    <script src="https://kit.fontawesome.com/9e84ade552.js" crossorigin="anonymous"></script>
</head>
<body>

<div class="wrapper">
    <div class="grid header">
        <div class="profile">
            <div class="name">
                <h1>Justin D Vrana</h1>
                <div class="tagline">
                    software engineer | synthetic biologist
                </div>
            </div>
            <div class="profilepic">
                <img src="https://secure.gravatar.com/avatar/34f83b6276a1d76d40684e503179e6cds?s=1000" />
            </div>
        </div>

    </div>

    <div class="grid content">
        <div class="about">
            Engineer and scientist that specializes in development of software that accelerates research and
            makes science more accessible. Avid self-learner with breadth of knowledge spanning from chemistry,
            cell biology, genomics, and software development. 14 years of wetlab experience and 6 years of
            software development experience.
        </div>
        <div class="work_experience">
            <h2>Relevant Experience</h2>

        <table class="experience">
            <tbody>
            <tr>
                <td>UW-Seattle<br>2018-2021</td>
                <td>Researcher</td>
                <td>
                    Graduate researcher for large-scale Synergistic Design and Discovery (SD2)
DARPA program effort to accelerate scientific research. Co-ordinated with
industry and academic teams across the U.S. to develop automated and
distributed design-build-test-learn pipeline for engineering new yeast strains.
Developed software-enabled cell construction pipeline which boosted yeast
cell strain construction throughput by 5-fold and reduced plasmid construction
labor and cost by 4-fold at campus cloud lab. Integrated software tools with
existing continuous integration (CI) and distributed computing infrastructure
from the Texas Advanced Computing Center (TACC).
                </td>
            </tr>
            <tr>
                <td>UW-Seattle<br>2014-2021</td>
                <td>Graduate Researcher</td>
                <td>Conducted synthetic biology research in yeast and mammalian cells. With
small-team, designed CRISPR technologies and tools for eukaryotic cells.
Performed mathematical modeling of cell behavior and fitted experimental
results using differential evolution optimization. Designed deep learning
models of cell circuit behavior using graph convolutional neural networks.
Developed deep reinforcement learning algorithms in Pytorch to design new
cell circuits from cell design specifications.</td>
            </tr>
             <tr>
                 <td>UW-Seattle<br>2018-2019</td>
                 <td>Teaching Assistant
                 <td>Taught programming in C/C++ for master's level Advanced Programming for
Embedded Systems course. Developed automated testing and grading scripts
using GoogleTest Framework for students.</td>
            </tr>
            <tr>
                <td>Global Impact<br>Seattle, WA<br>2017-2019</td>
                <td>Co-founder and Software Lead</td>
                <td>Co-founded small global health company around low-cost accessible
diagnostics (OLA-Simple). Developed iPad Ruby-on-Rails + AngularJS app to
guide nurses and clinical technician to perform low-cost HIV diagnostics for
patients based on Aquarium software. Won NSF I-Corp grant to conduct
customer research and business viability. Traveled Mexico City, Mexico and
Nairobi, Kenya to present research and technology to clinical and academic
researchers. Using app, 98% of new users were able successfully complete
diagnostics assay.</td>
            </tr>

            <tr>
                <td>UW-Seattle<br>2017-2018</td>
                <td>Amazon Catalyst Fellow</td>
                <td>Developed digitally guided protocols and workflows in which users are able to
design custom mammalian cell lines, develop experimental workflows to assay
them, view and analyze data, and execute experiments, all from the comfort of
a coffee shop. Awarded $100K from Amazon Catalyst program to develop
software and wetlab space.
                </td>
            </tr>

            <tr>
                <td>UC-Denver<br>Denver, CO<br>2011-2014</td>
                <td>Research Assistant</td>
                <td>
                    Conducted research in genetic engineering of yeast and mammalian cells.
Designed and performed experiments to validate light-activated genetic tools
for optogenetics research.
                </td>
            </tr>

            <tr>
                <td>UW-Madison<br>Madison, WI<br>2008-2011</td>
                <td>Undergraduate Research Assistant</td>
                <td>
                    Developed platform for single molecule dynamic studies and confined 2-
dimensional biochemistry. Used semi-automated microscope system and high-throughput computing cluster to construct low-resolution genome maps.
                </td>
            </tr>

<!--            <tr>-->
<!--                <td>2016-2018</td>-->
<!--                <td>Workflow Developer</td>-->
<!--                <td>UW Biofab, Seattle, Washington</td>-->
<!--                <td></td>-->
<!--            </tr>-->
<!--            <tr>-->
<!--                <td>2011-2014</td>-->
<!--                <td>Research Assistant</td>-->
<!--                <td>UC-Denver</td>-->
<!--                <td></td>-->
<!--            </tr>-->
<!--            <tr>-->
<!--                <td>2008-2011</td>-->
<!--                <td>Research Assistant</td>-->
<!--                <td>U of Wisconsin-Madison</td>-->
<!--                <td></td>-->
<!--            </tr>-->
            </tbody>
        </table>
        </div>
        <div class="portfolio_software">
            <h2>Selected Software Projects</h2>
        <table class="software_projects">
            <tbody>
                <tr>
                    <td>2019-present</td>
                    <td><a href="https://jvrana.github.io/caldera/">Caldera</a></td>
                    <td>Organism agnostic and data-driven genetic circuit designer driven by
                    deep graph neural network and reinforcement learning. Trains on single-cell
                    RNA-seq data and converts a behavioral specification to a genetic circuit
                    design.</td>
                </tr>

                <tr>
                    <td>2019-present</td>
                    <td><a href="https://github.com/jvrana/Terrarium">Terrarium</a></td>
                    <td>A dynamic computer-aided process planner (CAPP)
for biology designed for agile manufacturing of biological products. Converts a cell strain specifications into executable
                        wetlab plans.</td>
                </tr>
                <tr>
                    <td>2017-2020</td>
                    <td><a href="https://jvrana.github.io/DASi-DNA-Design/">DASi</a></td>
                    <td>DASi is an automatic DNA cloning plan designer aimed for operating on small budgets by
focusing on material re-use and a dead-simple user interface. Converts user-defined DNA sequences into executable molecular biology steps that builds specified sequences.</td>
                </tr>
                <tr>
                    <td>2016-2020</td>
                    <td><a href="https://www.aquarium.bio/">Aquarium</a></td>
                    <td>An open-source human-in-the-loop laboratory automation system that enables rapid,
flexible, and reproducible workflow development and execution.</td>
                </tr>
<!--                <tr>-->
<!--                    <td>2016-2020</td>-->
<!--                    <td>Trident</td>-->
<!--                    <td></td>-->
<!--                </tr>-->
            </tbody>
        </table>
        </div>
        <div class="portfolio_publications">
            <h2>Selected Publications</h2>
        <table class="publications">
          <tbody>
            <tr>
              <td>2021</td>
              <td>
                  <a href="https://academic.oup.com/synbio/advance-article/doi/10.1093/synbio/ysab006/6124325">
                      Aquarium: open-source laboratory software for design, execution and data management
                    </a>
              </td>
              <td>Oxford University Press: Synthetic Biology</td>
              <td>JD Vrana, OD Lange, ..., E Klavins</td>
            </tr>
            <tr>
              <td>2021</td>
              <td>Implementation of an interactive mobile application to pilot a rapid assay to detect HIV drug resistance mutations in Kenya</td>
              <td>AIDS<br><i>(accepted)</i></td>
              <td>JD Vrana, N Panpradist, ..., IA Beck</td>
            </tr>
<!--            <tr>-->
<!--              <td>2021</td>-->
<!--              <td>Simpler and faster Covid-19 testing: Strategies to streamline SARS-CoV-2 molecular assays</td>-->
<!--              <td>EBioMedicine</td>-->
<!--              <td>N Panpradist, Q Wang, PS Ruth, JH Kotnik, AK Oreskovic, A Miller, SWA Stewart,-->
<!--                    JD Vrana, PD Han, IA Beck, LM Starita, LM Frenkel, BR Lutz</td>-->
<!--            </tr>-->
            <tr>
                <td>2017</td>
                <td><a href="https://www.nature.com/articles/ncomms15459">
                    Digital logic circuits in yeast with CRISPR-dCas9 NOR gates</a>
                </td>
                <td>Nature Communications</td>
                <td>MW Gander, JD Vrana, WE Voje, JM Carothers, E Klavins</td>
            </tr>
<!--          <tr>-->
<!--              <td>2016</td>-->
<!--              <td>Benchmarking of optical dimerizer systems</td>-->
<!--              <td>ACS synthetic biology</td>-->
<!--              <td>GP Pathak, D Strickland, JD Vrana, CL Tucker</td>-->
<!--          </tr>-->
<!--            <tr>-->
<!--                <td>2014</td>-->
<!--                <td>Peripheral nervous system defects in a mouse model for peroxisomal biogenesis disorders</td>-->
<!--                <td>Developmental biology</td>-->
<!--                <td>MG Hanson, VL Fregoso, JD Vrana, CL Tucker, LA Niswander</td>-->
<!--          </tr>-->
            <tr>
              <td>2014</td>
              <td>
                  <a href="https://www.nature.com/articles/ncomms5925">
                      An optimized optogenetic clustering tool for probing protein interaction and function
                </a>
              </td>
              <td>Nature communications</td>
              <td>A Taslimi, JD Vrana, D Chen, S Borinskaya, BJ Mayer, MJ Kennedy, CL Tucker</td>
            </tr>
<!--            <tr>-->
<!--              <td>2014</td>-->
<!--              <td>Tools for controlling protein interactions using light</td>-->
<!--              <td>Current protocols in cell biology</td>-->
<!--              <td>CL Tucker, JD Vrana, MJ Kennedy</td>-->
<!--            </tr>-->
<!--            <tr>-->
<!--              <td>2013</td>-->
<!--              <td>Optogenetic control of cell function using engineered photoreceptors</td>-->
<!--              <td>Biology of the Cell</td>-->
<!--              <td>GP Pathak, JD Vrana, CL Tucker</td>-->
<!--            </tr>-->
<!--            <tr>-->
<!--              <td>2012</td>-->
<!--              <td>Rapid profiling of disease alleles using a tunable reporter of protein misfolding</td>-->
<!--              <td>Genetics</td>-->
<!--              <td>AMC Pittman, MD Lage, V Poltoratsky, JD Vrana, A Paiardini, A Roncador, CL Tucker</td>-->
<!--            </tr>-->
<!--            <tr>-->
<!--              <td>2012</td>-->
<!--              <td>Light-dependent, dark-promoted interaction between Arabidopsis cryptochrome 1 and phytochrome B proteins</td>-->
<!--              <td>Journal of Biological Chemistry</td>-->
<!--              <td>RM Hughes, JD Vrana, J Song, CL Tucker</td>-->
<!--            </tr>-->
          </tbody>
        </table>

<!--        <h3>Invited Talks</h3>-->
<!--        <table class="talks">-->
<!--          <tbody>-->
<!--            <tr>-->
<!--              <td>2019</td>-->
<!--              <td>Aquarium: a laboratory operating system for reproducible experimental design and execution</td>-->
<!--              <td>Society for Industrial Microbiology and Biotechnology, Automation session, Washington, D.C.</td>-->
<!--            </tr>-->
<!--          </tbody>-->
<!--        </table>-->
        </div>
    </div>

    <div class="grid sidebar">
        <div class="sidebar_contact">
            <h2 clas="underline">Contact</h2>
            <div class="flex2col">
            <div class="left">
                <ul>
                    <li>
                        <a href="mailto:justin.vrana@gmail.com"><i class="fas fa-envelope"></i> justin.vrana@gmail.com</a>
                    </li>
                    <li>
                        <a href="https://www.google.com/maps/place/Seattle,+WA+98125/@47.7170612,-122.3277998,13z/data=!3m1!4b1!4m5!3m4!1s0x54901166a8dcdd23:0x817006e4f7e1de9c!8m2!3d47.7170204!4d-122.3009337">
                            <i class="fas fa-map-marked"></i> 98125, Seattle, WA</a>
                    </li>
                    <li>
                        <i class="fas fa-phone"></i> (608) 234 3308
                    </li>
                    <li>
                        <a href="https://jvrana.github.io/markdown-cv/">
                            <i class="ai ai-cv ai-1x"></i> Website / CV
                        </a>
                    </li>
                </ul>
                </div>
            <div class="right">
                <ul>
                    <li>
                        <a href="https://scholar.google.com/citations?user=X5KqrDIAAAAJ&hl=en"><i class="ai ai-google-scholar ai-1x"></i> Google Scholar</a>
                    </li>
                    <li>
                        <div itemscope itemtype="https://schema.org/Person"><a itemprop="sameAs" content="https://orcid.org/0000-0002-0862-0824" href="https://orcid.org/0000-0002-0862-0824" target="orcid.widget" rel="me noopener noreferrer" style="vertical-align:top;">
                            <i class="ai ai-orcid ai-1x"></i> ORDIC ID</a></div>
                    </li>
                    <li>
                        <a href="https://www.researchgate.net/profile/Justin-Vrana"><i class="ai ai-researchgate ai-1x"></i> Research Gate</a>
                    </li>
                    <li>
                        <a href="https://www.github.com/jvrana"><i class="fab fa-github"></i> github.com/jvrana</a><br>
                    </li>
                    <li>
                        <a href="https://www.linkedin.com/in/justin-vrana-bioe/"><i class="fab fa-linkedin-in"></i> LinkedIn</a><br>
                    </li>
                </ul>
            </div>
        </div>
        </div>

<!--        <div class="sidebar_contact">-->
<!--            <h2 class="underline">Contact</h2>-->
<!--                <b>Address</b>-->
<!--                <br>-->
<!--                <a href="https://www.google.com/maps/place/Seattle,+WA+98125/@47.7170612,-122.3277998,13z/data=!3m1!4b1!4m5!3m4!1s0x54901166a8dcdd23:0x817006e4f7e1de9c!8m2!3d47.7170204!4d-122.3009337">98125, Seattle, WA</a>-->
<!--                <br>-->

<!--                <br>-->
<!--                <b>Github</b>-->
<!--                <br>-->
<!--                <a href="https://www.github.com/jvrana">github.com/jvrana</a>-->
<!--                <br>-->

<!--                <br>-->
<!--                <b>Email</b>-->
<!--                <br>-->
<!--                <a href="mailto:justin.vrana@gmail.com">justin.vrana@gmail.com</a>-->
<!--                <br>-->

<!--                <br>-->
<!--                <b>Web</b><br><a href="https://jvrana.github.io/markdown-cv/">https://jvrana.github.io/markdown-cv/</a>-->
<!--                <br>-->

<!--                <br>-->
<!--                <b>Social Media</b>-->
<!--                </br>-->
<!--                <a href="https://scholar.google.com/citations?user=X5KqrDIAAAAJ&hl=en">Google Scholar</a>-->
<!--                <div itemscope itemtype="https://schema.org/Person"><a itemprop="sameAs" content="https://orcid.org/0000-0002-0862-0824" href="https://orcid.org/0000-0002-0862-0824" target="orcid.widget" rel="me noopener noreferrer" style="vertical-align:top;"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" style="width:1em;margin-right:.5em;" alt="ORCID iD icon">ORDIC ID</a></div>-->
<!--                <a href="https://www.linkedin.com/in/justin-vrana-bioe/">LinkedIn</a><br>-->
<!--                <a href="https://www.researchgate.net/profile/Justin-Vrana">Research Gate</a>-->
<!--        </div>-->
        <div class="sidebar_education">
            <h2>Education</h2>
            <div class="education">
                <div class="date">
                    2021 (anticpated)
                </div>
                <div class="degree">
                    PhD, Bioengineering
                </div>
                <div class="school">
                    University of Washington-Seattle
                </div>

                <div class="date">
                   2010
                </div>
                <div class="degree">
                    BS, Chemistry & Philosophy
                </div>
                <div class="school">
                    University of Wisconsin-Madison
                </div>


                <div class="date">
                    2010
                </div>
                <div class="degree">
                    Cert. of Computer Science
                </div>
                <div class="school">
                    University of Wisconsin-Madison
                </div>
            </div>
        </div>
        <div class="sidebar_skills">
            <h2>Skills</h2>
            <div style="margin-top: 30px">
                <h3><i class="fas fa-code"></i> Software</h3>

                <ul>
                    <li>
                        <b>Languages</b> Python (expert), Javascript, Ruby, C, C++, Java (novice)
                    </li>
                    <li>
                        <b>Web Frameworks</b> Web Frameworks Ruby-on-Rails, Flask, Django, React, GraphQL
                    </li>
                    <li>
                        <b>Databases</b> SQL, no-SQL mongoDB, and noSQL Neo4j (graph database)
                    </li>
                    <li>
                        <b>Machine Learning</b> Deep reinforcement learning & graph neural networks using Pytorch and TensorFlow
                    </li>
                    <li>
                        <b>Visualization</b> Python tools such as matplotlib/seaborn, Plotly, Bokeh, Jupyter, D3.js (novice)
                    </li>
                    <li>
                        <b>Team Co-ordination</b>
                        Proficient with common team tools such as Jira, Asana, Slack.
                        Proficient with continous-integration and development tools on Docker, DockerCompose, Dockerhub, Git, Gitlab, Github, CircleCI, and Jenkins.
                    </li>
Web Frameworks Ruby-on-Rails, Flask,
Django, React, GraphQL
Databases SQL, no-SQL mongoDB, and noSQL Neo4j (graph database)
Machine Learning Deep reinforcement
learning & graph neural networks using
Pytorch and TensorFlow
Visualization Python tools such as
matplotlib/seaborn, Plotly, Bokeh, Jupyter.
D3.js (novice)
Team Co-ordination Proficient with common
team tools suchs as Jira, Asana, Slack.
Proficient with connuous-integraon and
development tools on Docker, Dockercompose, Dockerhub, Git, Gitlab, Github,
CircleCI, and Jenkins.
Other skills discrete mathemacs
(networks/graphs), algorithm development
                </ul>
                <b>Languages</b>
                Python (expert), Javascript, Ruby, C, C++, Java
                <br>
                <b>Frameworks</b>
                Ruby-on-Rails, Flask, React, Pytorch, Docker, SQL, no-SQL (mongoDB & Neo4j)
                <br>
                <b>Machine Learning</b>
                deep reinforcement learning, graph neural networks
                <br>
                <b>Other skills</b>
                discrete mathematics (networks/graphs), algorithm development
            </div>
            <div style="margin-top: 30px">
                <h3><i class="fas fa-flask"></i> Wetlab</h3>
                <b>Mammalian</b>
                lentiviral transductions, DNA transfections, cell line maintainance, microscopy, cell sorting, flow cytometry
                <br>
                <b>Microbiology</b>
                CRISPR cell engineering, yeast genomic integrations, plasmid construction, DNA cloning
            </div>


        </div>
<!--        <div class="sidebar_awards">-->
<!--            <h2>Selected Awards</h2>-->
<!--            <table class="awards">-->
<!--                <tbody>-->
<!--                    <tr>-->
<!--                        <td>2018</td>-->
<!--                        <td>DLA Piper $2,500 Best Idea with Global Reach</td>-->
<!--                        <td>A start up centered around a drug-resistance detection diagnostic automation</td>-->
<!--                    </tr>-->
<!--                    <tr>-->
<!--                        <td>2018</td>-->
<!--                        <td>UW Business Plan Competition (Final 16)</td>-->
<!--                        <td>For business plan for OLASimple.</td>-->
<!--                    </tr>-->
<!--                    <tr>-->
<!--                        <td>2017</td>-->
<!--                        <td>$100K Amazon Catalyst Grant</td>-->
<!--                        <td>For cloud laboratory for genetic engineering to expand to automated-->
<!--            mammalian cell work.</td>-->
<!--                    </tr>-->
<!--    &lt;!&ndash;                <tr>&ndash;&gt;-->
<!--    &lt;!&ndash;                    <td>2015</td>&ndash;&gt;-->
<!--    &lt;!&ndash;                    <td>CSN Sandbox Competition</td>&ndash;&gt;-->
<!--    &lt;!&ndash;                    <td>For VertiGone, an app & hardware that detects the onset of a fall and helps reorient patients using 3D auditory cues.</td>&ndash;&gt;-->
<!--    &lt;!&ndash;                </tr>&ndash;&gt;-->
<!--                </tbody>-->
<!--            </table>-->
<!--        </div>-->
    </div>
</div>
</body>
</html>